LayoutTests/platform/ios/TestExpectations

```diff
@@ -3324,6 +3324,10 @@ imported/w3c/web-platform-tests/css/css-lists/list-style-type-string-002.html [
 # These tests are for iOS port. Should be pass.
 fast/shadow-dom/ios/ [ Pass ]
 
+# This test fails in iOS due to subpixel differences for the moongolian characters
+imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-normal.html [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-strict.html [ ImageOnlyFailure ]
```

Comment:
if these failures here really just about subpixel diffs, it may be better to introduce fuzziness

Reply:
These are WPT tests and if I add even 'fuzziness' meta tag, on next import, it will be overridden.

Reply:
I am pretty sure there's a way to avoid it, we do it all the time.

Reply:
Cool! I never knew it. Good to know. Will add meta tag for pixel tolerance. 👍

Reply:
grep -i -r "meta name=\"fuzzy" LayoutTests/imported/w3c/web-platform-tests/css -> 425